ios_app_clip/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -484,15 +506,9 @@
 				CLANG_WARN_UNGUARDED_AVAILABILITY = YES_AGGRESSIVE;
 				CODE_SIGN_ENTITLEMENTS = AppClip/AppClip.entitlements;
 				CODE_SIGN_STYLE = Automatic;
-				DEVELOPMENT_TEAM = "";
 				ENABLE_BITCODE = NO;
-				FRAMEWORK_SEARCH_PATHS = (
```

Comment:
I removed this instructions from https://docs.flutter.dev/platform-integration/ios/ios-app-clip#step-7---integrate-flutter in https://github.com/flutter/website/pull/10145, it's not necessary.